core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -62,21 +62,24 @@ object BootMonitors extends LazyLogging {
     //Boot submission monitor supervisor
     val submissionmonitorConfigRoot = conf.getConfig("submissionmonitor")
     val submissionMonitorConfig = SubmissionMonitorConfig(util.toScalaDuration(submissionmonitorConfigRoot.getDuration("submissionPollInterval")), submissionmonitorConfigRoot.getBoolean("trackDetailedSubmissionMetrics"))
-    startSubmissionMonitorSupervisor(
-      system,
-      submissionMonitorConfig,
-      dataSourceAccess,
-      samDAO,
-      gcsDAO,
-      shardedExecutionServiceCluster,
-      metricsPrefix
-    )
+//    startSubmissionMonitorSupervisor(
```

Comment:
Ignore this, just commented out because I wanted a back rawls locally but didn't want all of these other monitors spamming my logs